File: CartService/Models/Events/PaymentSuccessEvent.cs
public class PaymentSuccessEvent
{
    public int PaymentId { get; set; }
    public int OrderId { get; set; }
    public int UserId { get; set; }
    public double Amount { get; set; }
    public string Status { get; set; } = string.Empty;
    public DateTime PaymentDate { get; set; }
}

File: order-service/src/main/java/org/example/orderservice/dto/PaymentSuccessEvent.java
package org.example.orderservice.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
public class PaymentSuccessEvent {
    @JsonProperty("PaymentId")
    private int paymentId;

    @JsonProperty("OrderId")
    private int orderId;

    @JsonProperty("UserId")
    private int userId;

    @JsonProperty("Amount")
    private double amount;

    @JsonProperty("Status")
    private String status;

    @JsonProperty("PaymentDate")
    private LocalDateTime paymentDate;
}
